packages/app/lib/betterSqlite3Connection.ts

```diff
@@ -0,0 +1,64 @@
+import type { Schema } from '@tloncorp/shared/dist/db';
+import type { AnySqliteDatabase } from '@tloncorp/shared/dist/db/client';
+import { migrations } from '@tloncorp/shared/dist/db/migrations';
+import type { Database } from 'better-sqlite3';
+import type { DrizzleConfig } from 'drizzle-orm';
+import { drizzle } from 'drizzle-orm/better-sqlite3';
+
+import type { SQLiteConnection } from './sqliteConnection';
+
+export class BetterSqlite3$SQLiteConnection implements SQLiteConnection {
+  constructor(private connection: Database) {}
+
+  execute(query: string): void {
+    this.connection.exec(query);
+  }
+
+  updateHook(
+    _callback:
+      | ((params: {
+          table: string;
+          operation: 'INSERT' | 'DELETE' | 'UPDATE';
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          row?: any;
+          rowId: number;
+        }) => void)
+      | null
+  ): void {
+    console.warn(
+      'BetterSqlite3$SQLiteConnection::updateHook() is not implemented'
+    );
```

Comment:
We're not testing code that response to subscriptions, so this isn't an issue – but I figured it'd be annoying to try to add a test that relies on `updateHook` and not get a big warning that it's not yet implemented.